app/views/mail.scala.html

```diff
@@ -0,0 +1,48 @@
+<table class="mailtable" style="width: 100%;">
+    <tr>
+        <td>
+            <table class="contenttable" style="width: 90%;margin: auto;">
+                <tr>
+                    <td class="logobox" style="text-align: center;"><img class="logo" src="https://media.licdn.com/media/AAEAAQAAAAAAAAKKAAAAJGYyN2NhODQ4LTQ5MTYtNGEwYy05NjA0LTAwOTNiMDNjNmNhOQ.png"/>
+                        <br>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        &nbsp;
+                    </td>
+                </tr>
+                <tr>
+                    <td class="titlebox" style="background-color: rgba(0, 117, 162, 0.4);padding: 20px;">
+                        <span class="titleboxtitle" style="font-family: Arial;font-size: 20px;font-weight: bold;color: #000;">Friday Lunch !!!</span>
+                        <br><p></p>
+                        <span class="titleboxtext" style="font-family: Arial;font-size: 14px;color: #000;">It's a gentle reminder to to fill your availablity <a href="http://lunatech-lunch.cleverapps.io/" style="font-size: 18px;font-weight: bold;">Lunatech-lunch</a>.
+							<br>
+							Please fill your food preferences also. </span>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        &nbsp;
+                    </td>
+                </tr>
+                <tr>
+                    <td class="titlebox2" style="background-color: rgba(213, 86, 114, 0.4);padding: 20px;">
+                        <br><p></p>
+                        <span class="titleboxtext" style="font-family: Arial;font-size: 14px;color: #000;">
+                            <li>If you have any suggestions please mail us at developers@@lunatech.com</li>
```

Comment:
I don't know that we actually have a developer@lunatech.com. If its not, Can you please suggest which email address we should use here.

yes we have to use two @'s because in scala template one '@' treated as scala syntax but if we want to simply render '@' symbol on our template then we can use two @'s.